packages/ui/src/components/hover-card.tsx

```diff
@@ -27,7 +40,8 @@ HoverCardContent.displayName = HoverCardPrimitive.Content.displayName
 const HoverCard = {
   Root: HoverCardRoot,
   Trigger: HoverCardTrigger,
-  Content: HoverCardContent
+  Content: HoverCardContent,
+  Arrow: HoverCardArrow
```

Comment:
I wonder if we have a use-case where we _don't_ want the arrow? Unless there is that I don't know about, for consistency, I'd bake the arrow into the `Content` component and not export it here at all.

Reply:
Currently, the component is used once, and we don't actually know if we need the arrow as a separate component, so it makes sense to do what you suggested 👍